aep/general/0154/aep.yaml

```
@@ -1,7 +1,8 @@
 ---
 id: 154
 state: approved
+slug: etag
```

```suggestion
slug: etags
```

WDYT?  Other slugs are pluralized where applicable.